lectures/cagan_ree.md

```diff
@@ -18,13 +18,10 @@ kernelspec:
 
 We'll use linear algebra first to explain and then do  some experiments with  a "monetarist theory of  price levels".
 
+Economists call it a "monetary" or "monetarist" theory of  price levels because  effects on  price levels occur via a central banks's  decisions to print  money supply.  
```

Comment:
@HumphreyYang this is a minor thing -- but do you mind removing the `multiple spaces` as you work through these documents. It doesn't impact visually but it's a good thing to do I think.